pkg/crc/machine/stop.go

```diff
@@ -19,7 +19,7 @@ func (client *client) Stop() (state.State, error) {
 		return state.Error, errors.Wrap(err, "Cannot load machine")
 	}
 	if err := stopAllContainers(host, client); err != nil {
-		return state.Error, err
+		logging.Warnf("Failed to stop all containers: %v\nProceeding with stop...", err)
```

Comment:
maybe `Failed to stop OpenShift containers` (or cluster containers). `Proceeding with VM shutdown`, this is more accurate about what is being stopped, and what these containers are. I'd favour only having `logging.Debugf("%v", err)` for the error printing, and not include it in the `logging.Warnf`.

Reply:
This comment seems to have been missed before this PR got merged.